### ISSUE TITLE
opam info for tags is hard to parse

### ISSUE THREAD

**Issue by avsm** [ocaml/opam#3365]
```
$ opam show -f tags: root1d
"scientfic" "root finding"
$ opam show -f tags: mirage-xen-ocaml
org:mirage
```

This is quite hard to parse from OCaml -- sometimes the output is string-quoted (and `%S` from `scanf` works) and sometimes the output is bare.  It would be useful to make this a bit more systematic before 2.0 so that it can be depended on for use in shell and script output.

**Comment by avsm** [comment]
This is the rough OCaml code I'm using so far:

```
let parse_string_list buf =
  let open Scanf in
  let open Astring in
  match String.find (function ' ' | '"' -> true | _ -> false) buf with
  | Some _ ->
     let rec fn ic = kscanf ic (fun _ _ -> []) "%S %r" fn (fun a b -> a::b) in
     fn (Scanning.from_string buf)
  | None -> [buf]
```

**Comment by AltGr** [comment]
Sorry I missed it earlier, but there is also the `--normalise` option. Not sure it's perfect for your purpose, but it will yield:
```
% opam show -f tags: mirage-xen-ocaml --normalise
"org:mirage"
% opam show -f tags: root1d --normalise          
["scientfic" "root finding"]
```
so you still have to deal with the optional brackets, but at least the quoting is consistent.
